README.md

```diff
@@ -1,3 +1,161 @@
-# Run Parasoft SOAtest action
+# Run Parasoft SOAtest
 
-This action enables you to run functional tests with Parasoft SOAtest and review results on GitHub.
\ No newline at end of file
+[![Build](https://github.com/parasoft/run-soatest-action/actions/workflows/build.yml/badge.svg?branch=master)](https://github.com/parasoft/run-soatest-action/actions/workflows/build.yml)
+
+This action enables you to run static analysis and execute tests with Parasoft SOAtest and review results on GitHub.
+
+Parasoft SOAtest is an enterprise-grade solution that simplifies complex testing for business-critical transactions through APIs, message brokers, databases mainframes, ERPs, browser-based UIs, and other endpoints. SOAtest helps QA teams ensure secure, reliable, compliant business applications with an intuitive interface to create, maintain, and execute end-to-end testing scenarios.
+- Request [a free trial](https://www.parasoft.com/products/parasoft-soatest/soatest-request-a-demo/) to receive access to Parasoft SOAtest's features and capabilities.
+- See the [user guide](https://docs.parasoft.com/display/SOA20232) for information about Parasoft SOAtest's capabilities and usage.
+
+Please visit the [official Parasoft website](http://www.parasoft.com) for more information about Parasoft SOAtest and other Parasoft products.
+
+## Quick start
+
+To run your Parasoft SOAtest and publish test results on GitHub, you need to customize your GitHub workflow to include:
```

Comment:
Please change `publish` to `review` since we are not publishing the results as an special artifact.